google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc

```diff
@@ -162,17 +164,25 @@ int main(int argc, char* argv[]) {
             << ",RowCount,ElapsedTime,CpuTime,StatusCode\n"
             << std::flush;
 
-  auto experiment = e->second;
-  experiment->SetUp(config, database);
-  experiment->Run(generator, config, database);
-  experiment->TearDown(config, database);
+  int exit_status = EXIT_SUCCESS;
+
+  auto experiment = e->second(generator);
+  auto status = experiment->SetUp(config, database);
+  if (!status.ok()) {
+    std::cout << "# Skipping experiment, SetUp() failed\n";
```

Comment:
Opt: Include `status` in message (like in "run-all")?  Perhaps `SetUp()` already logged it, but perhaps it shouldn't?

Reply:
Done.